### DISCUSSION TITLE
group 2 useful links

### DISCUSSION THREAD

**Discussion by aviv82** [lab-antwerp-1/home#8] [Helpful-materials]
group 2!

we have a lot of work ahead of us, and not a lot of time. we can do a great job as long as we are organized. 
i am starting this discussion to share a few resources to help us do it well. 

also we can check in here and let each other know what we are working on and how its coming along.

here are a couple of useful links on setting ssh keys git, github and branching: 

https://github.com/lab-antwerp-1/home/discussions/6   <-- how to set up ssh key.
https://www.youtube.com/watch?v=RGOj5yH7evk&t=3391s  <-- this video is long, but super helpful. especially the part about branching. 

a helpful video on how to write markdown: 
https://www.youtube.com/watch?v=eJojC3lSkwg&t=456s

we will have a meeting wednesday (6th) morning and hopefully have the main README.md done by then, so we can focus the rest of the time on our individual intro files.

**Comment by aviv82** [comment]
today i will work on my branch of the README.md and try to make it nice looking and well written.

**Reply by Mervatabuamro** [reply]
ok great I will look  through some links today to better understand some issues and we can meet tomorrow at 10 am that is ok for me

**Reply by aviv82** [reply]
sounds good. i accidentally started this discussion in the wrong repo... so i'm moving it to our group's repo:

https://github.com/lab-antwerp-1/group-2/discussions/1
